contrib/scripts/localnet-uniswap.sh

```
@@ -0,0 +1,261 @@
+# nibid config
+echo_info "Updating nibid config..."
+$BINARY config keyring-backend test
+$BINARY config chain-id $CHAIN_ID
+$BINARY config broadcast-mode sync
+$BINARY config output json
+$BINARY config node "http://localhost:26657"
+$BINARY config # Prints config.
+
+# Enable API Server
+echo_info "config/app.toml: Enabling API server"
+sed -i $SEDOPTION '/\[api\]/,+3 s/enable = false/enable = true/' $CHAIN_DIR/config/app.toml
```

Shellcheck (0.10.0)

[warning] Quotes/backslashes in this variable will not be respected.

(SC2090)